pkg/service/v0/accounts.go

```diff
@@ -320,6 +323,17 @@ func (s Service) CreateAccount(c context.Context, in *proto.CreateAccountRequest
 		return
 	}
 
+	// TODO: All admin for now as create Account request does not have any role
+	rs := settings.NewRoleService("com.owncloud.api.settings", mgrpc.NewClient())
+	_, err = rs.AssignRoleToUser(c, &settings.AssignRoleToUserRequest{
+		AccountUuid: acc.Id,
+		RoleId:      settings_svc.BundleUUIDRoleAdmin,
```

Comment:
Please use the user role `BundleUUIDRoleUser`. We'll create one new admin user in the set of default users (einstein, marie, etc). All others should be regular users until we decide something else.